src/platform/forms-system/src/js/web-component-fields/VaFileInputField.jsx

```diff
@@ -43,20 +47,14 @@ const VaFileInputField = props => {
   const { fileUploadUrl } = mappedProps;
 
   useEffect(() => {
-    const { localFilePath } = props.childrenProps.formData;
-    if (localFilePath) {
-      const fetchFile = async () => {
-        await fetch(localFilePath)
-          .then(r => r.blob())
-          .then(blob =>
-            setLocalFile(
-              new File([blob], props.childrenProps.formData.name, {
-                type: 'application/pdf',
-              }),
-            ),
-          );
-      };
-      fetchFile();
+    const localFileAsString = localStorage.getItem('uploadedFile');
```

Comment:
what was the concern with URL.createObjectURL again? That seemed like a good solution. It cleans up automatially when the windows is closed and seemed appropriate for PDFs.  definitely don't want `localStorage`, because that is retained even after the browser is closed. Possible `sessionStorage` could be an alternative.